src/dashboard/cd-dashboard-projects.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <div class="cd-dashboard-projects">
-    <h3 class="cd-dashboard-projects__header">Before your next event, here are some projects you can try</h3>
+    <h2 class="cd-dashboard-projects__header hidden-xs">Before your next event, here are some projects you can try</h2>
+    <h2 class="cd-dashboard-projects__header visible-xs">Here are some projects you can try</h2>
```

Comment:
Hum, why this reduction of the sentence? To look like a one-liner ?

Reply:
That's what was in de mobile template. Also bc it looks awkwardly long as a header in mobile imo

Reply:
Then add trnaslations as well :)